### PR TITLE
Major cleanup to concentrate on being an actual guide

### PR DESCRIPTION
**Pull request by karras**
This PR removes quite a lot of Ansible and Vagrant related stuff. The idea of this repository should be to provide a guide and only a guide. I extended the makefile to automatically generate the required requirements.yml file to ensure that all our roles on galaxy are automatically added to the docs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,10 +11,4 @@
...
 
 # documentation
-/doc/_build/
-/doc/*.yml.rst
-/doc/role_overview.rst
-/doc/sphinx-template/
-
-# vagrant
-/.vagrant/
+/doc/
```
Comment (keachi) [inline .gitignore]: Why do you ignore the whole `/doc/`? There are some required files (e.g. `index.rst`). The old configuration just ignored the generated documents.

**Review by keachi — CHANGES_REQUESTED** [review]
Just the gitignore

File: Makefile [hunk 2]
```diff
@@ -10,17 +10,23 @@ help:  ## display this help
...
 		make clean; \
 		cd ..; \
 	)
-	vagrant destroy -f
 
-test:  ## run test environment
-	vagrant up --provision
+requirements: ## create requirements.yml
+	echo "---" > requirements.yml
+	for role in $$(ansible-galaxy search --author adfinis-sygroup | grep adfinis | cut -d ' ' -f2); do \
```
Comment (hairmare) [inline Makefile]: You could lose a `|` by using awk:
```suggestion
	for role in $$(ansible-galaxy search --author adfinis-sygroup | awk '/adfinis/ {print $2}'); do \
```

File: .gitignore (hunk 1 above)
Comment (karras) [reply .gitignore]: Fixed and squashed, git horray!

File: Makefile (hunk 2 above)
Comment (karras) [reply Makefile]: Fixed and rebased. Note that it has to be the first variable though. ;)

**Comment by karras** [comment]
Damn, Travis prints the `-e` parameter of echo, meanwhile it works on my machine :(

```
-e - src: adfinis-sygroup.ansible
  version: master
```

File: Makefile (hunk 2 above)
Comment (hairmare) [reply Makefile]: yeah, I don't really `cut` by heart like I `awk`

File: doc/conf.py [hunk 3]
```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'Ansible Guide'
-copyright = '2017, Adfinis SyGroup AG'
+copyright = '2019, Adfinis SyGroup AG'
```
Comment (hairmare) [inline doc/conf.py]:
```suggestion
copyright = '2017 - 2019, Adfinis SyGroup AG'
```
?

Or (my personal fave):
```suggestion
copyright = 'Adfinis SyGroup AG'
```

or even more python:
```suggestion
copyright = '2017 - {}, Adfinis SyGroup AG'.format(datetime.datetime.now().year)
```

File: Makefile [hunk 4]
```diff
@@ -10,17 +10,23 @@ help:  ## display this help
...
 		cd ..; \
 	)
-	vagrant destroy -f
 
-test:  ## run test environment
-	vagrant up --provision
+requirements: ## create requirements.yml
+	echo "---" > requirements.yml
+	for role in $$(ansible-galaxy search --author adfinis-sygroup | awk '/adfinis/ {print $$1}'); do \
+	    echo -e "- src: $$role\n  version: master" >> requirements.yml; \
```
Comment (hairmare) [inline Makefile]: printf or tee? 🤷‍♂️

File: doc/conf.py (hunk 3 above)
Comment (karras) [reply doc/conf.py]: Good point, I don't like to maintain such stuff neither. :)

File: Makefile (hunk 4 above)
Comment (karras) [reply Makefile]: Fixed, ty kind sir

**Comment by karras** [comment]
@keachi Please approve if OK.

**Review by keachi — APPROVED** [review]

**Pull request merged**